examples/oneapi/cpp/source/jaccard/jaccard_batch.cpp

```diff
@@ -14,37 +14,38 @@
  * limitations under the License.
  *******************************************************************************/
 
+
 #include "oneapi/dal/algo/jaccard.hpp"
-#include <iostream>
-#include "oneapi/dal/data/graph.hpp"
+#include "oneapi/dal/data/undirected_adjacency_array_graph.hpp"
 #include "oneapi/dal/data/table.hpp"
```

Comment:
are there any rules on the ordering of headers
What dal examples include, only onedal.hpp or a set of headers

Reply:
Fixed and aligned with ML examples